pkgs/development/compilers/elm/packages/elm-format.nix

```
@@ -1,40 +1,40 @@
-{ mkDerivation, fetchgit, ansi-terminal, ansi-wl-pprint, array, base, binary
-, bytestring, cmark, containers, directory, filepath, free, HUnit
-, indents, json, mtl, optparse-applicative, parsec, process
-, QuickCheck, quickcheck-io, split, lib, tasty, tasty-golden
-, tasty-hunit, tasty-quickcheck, text
+{ mkDerivation, ansi-wl-pprint, avh4-lib, base, bimap, cmark
+, containers, elm-format-lib, elm-format-test-lib, fetchgit, json
+, lib, mtl, optparse-applicative, parsec, QuickCheck, quickcheck-io
+, relude, tasty, tasty-hspec, tasty-hunit, tasty-quickcheck, text
 }:
 mkDerivation {
   pname = "elm-format";
-  version = "0.8.4";
+  version = "0.8.5";
   src = fetchgit {
     url = "https://github.com/avh4/elm-format";
-    sha256 = "0cxlhhdjx4h9g03z83pxv91qrysbi0ab92rl52jb0yvkaix989ai";
-    rev = "5bd4fbe591fe8b456160c180cb875ef60bc57890";
+    sha256 = "0bcjkcs1dy1csz0mpk7d4b5wf93fsj9p86x8fp42mb0pipdd0bh6";
+    rev = "80f15d85ee71e1663c9b53903f2b5b2aa444a3be";
+    fetchSubmodules = true;
   };
-  postPatch = ''
-    mkdir -p ./generated
-    cat <<EOHS > ./generated/Build_elm_format.hs
-    module Build_elm_format where
-
-    gitDescribe :: String
-    gitDescribe = "0.8.4"
-    EOHS
-  '';
   isLibrary = false;
   isExecutable = true;
-  libraryHaskellDepends = [
-    ansi-terminal ansi-wl-pprint array base binary bytestring
-    containers directory filepath free indents json mtl
-    optparse-applicative parsec process split text
+  executableHaskellDepends = [
+    ansi-wl-pprint avh4-lib base containers elm-format-lib json
+    optparse-applicative relude text
   ];
-  executableHaskellDepends = [ base ];
   testHaskellDepends = [
-    base cmark containers HUnit mtl parsec QuickCheck quickcheck-io
-    split tasty tasty-golden tasty-hunit tasty-quickcheck text
+    ansi-wl-pprint avh4-lib base bimap cmark containers elm-format-lib
+    elm-format-test-lib json mtl optparse-applicative parsec QuickCheck
+    quickcheck-io relude tasty tasty-hspec tasty-hunit tasty-quickcheck
+    text
   ];
   doHaddock = false;
   homepage = "https://elm-lang.org";
   description = "A source code formatter for Elm";
   license = lib.licenses.bsd3;
+  postPatch = ''
+    mkdir -p ./generated
+    cat <<EOHS > ./generated/Build_elm_format.hs
+    module Build_elm_format where
+
+    gitDescribe :: String
+    gitDescribe = "0.8.5"
```

and then (with `rec`) refer to `version` attribute here:

```suggestion
    gitDescribe = ${version}
```